community/server/src/test/java/org/neo4j/server/integration/StartupLoggingIT.java

```diff
@@ -0,0 +1,100 @@
+package org.neo4j.server.integration;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import org.neo4j.server.CommunityBootstrapper;
+import org.neo4j.test.SuppressOutput;
+import org.neo4j.test.server.ExclusiveServerTestBase;
+
+import static java.util.Arrays.asList;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class StartupLoggingIT extends ExclusiveServerTestBase
+{
+    @Test
+    public void shouldLogHelpfulStartupMessages() throws Throwable
+    {
+        // Given
+        SuppressOutput suppressed = SuppressOutput.suppressAll();
+
+        // When
+        suppressed.call( new Callable<Object>()
+        {
+            @Override
+            public Object call() throws Exception
+            {
+                CommunityBootstrapper boot = new CommunityBootstrapper();
+                CommunityBootstrapper.start( boot, new String[]{} );
+                boot.stop();
+                return null;
+            }
+        });
+
+        // Then
+        List<String> captured = suppressed.getOutputVoice().lines();
+        assertThat( captured, matchesLines(
+                info( "Neo4j is starting.." ),
+                info( "Neo4j has started! The HTTP API and the Neo4j Browser are now available at http://.*:7474/" ),
```

Comment:
We shouldn't change this output in 2.3, as people do log scraping for existing values.